### ISSUE TITLE
getting crash clicking run in editor checkbox a few times

### ISSUE THREAD

**Issue by slapin** [Zylann/godot_voxel#313]
Clicked run in editor button 2 times and got this. I use gdscript as generator (prototyping, with "tool" on top) so this might be something to do with this... it is effectively heightmap though. This happens only occasionally.

```
ERROR: FATAL: Condition "!lod.blocks_to_load.empty()" is true.
   at: _process (/home/slapin/godot-projects/small-game/src/modules/voxel/terrain/voxel_lod_terrain.cpp:1068)

================================================================
handle_crash: Program crashed with signal 4
Engine version: Godot Engine v3.4.beta.custom_build (316b5dad2dfcd8758372b728fb059b067b57650c)
Dumping the backtrace. Please include this when reporting the bug on https://github.com/godotengine/godot/issues
[1] /lib/x86_64-linux-gnu/libc.so.6(+0x3be30) [0x7fcd38eaee30] (??:0)
[2] ../../src/godot/bin/godot.x11.opt.tools.64() [0xcf81fd] (/home/slapin/godot-projects/small-game/src/modules/voxel/terrain/voxel_lod_terrain.cpp:1068 (discriminator 1))
[3] ../../src/godot/bin/godot.x11.opt.tools.64() [0x3264d34] (/home/slapin/godot-projects/small-game/src/godot/core/object.cpp:929)
[4] ../../src/godot/bin/godot.x11.opt.tools.64() [0x22dd35a] (/home/slapin/godot-projects/small-game/src/godot/scene/main/scene_tree.cpp:977)
[5] ../../src/godot/bin/godot.x11.opt.tools.64() [0x22e7860] (/home/slapin/godot-projects/small-game/src/godot/scene/main/scene_tree.cpp:528 (discriminator 3))
[6] ../../src/godot/bin/godot.x11.opt.tools.64() [0x7da03f] (/home/slapin/godot-projects/small-game/src/godot/main/main.cpp:2184)
[7] ../../src/godot/bin/godot.x11.opt.tools.64() [0x7be211] (/home/slapin/godot-projects/small-game/src/godot/platform/x11/os_x11.cpp:3641)
[8] ../../src/godot/bin/godot.x11.opt.tools.64() [0x7a6b55] (/home/slapin/godot-projects/small-game/src/godot/platform/x11/godot_x11.cpp:55)
[9] /lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0xea) [0x7fcd38e99cca] (??:0)
[10] ../../src/godot/bin/godot.x11.opt.tools.64() [0x7ac47a] (??:?)
-- END OF BACKTRACE --
================================================================
```

**Comment by Zylann** [comment]
Pushed a fixed in 0838cab251b3dd08670ea0bd8eaaf999c2ab3cc8, let me know if it works for you

**Comment by slapin** [comment]
With latest patch the problem is no longer reproducible, thanks!
